.github/workflows/lighthouse_accessibility.yml

```diff
@@ -25,46 +25,22 @@ jobs:
           python-version: '3.8.15'
           architecture: 'x64'
       - uses: ./.github/actions/merge
-      # Caching is disabled to avoid poisoning our cache with the
-      # changed lighthouse file.
-      #
-      # - name: Cache node modules
-      #   uses: actions/cache@v3
-      #   env:
-      #     cache-name: cache-node-modules
-      #   with:
-      #     path: /home/runner/work/oppia/yarn_cache
-      #     key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('yarn.lock') }}
-      #     restore-keys: |
-      #       ${{ runner.os }}-build-${{ env.cache-name }}-
-      #       ${{ runner.os }}-build-
-      #       ${{ runner.os }}-
-      - uses: ./.github/actions/install-oppia-dependencies
-      - run: |
-          # Replace the 1.5 second timeout in lighthouse for retrieving
-          # sourcemaps with a 10 second timeout. This timeout is not
-          # configurable through Lighthouse's exposed API, so we have
-          # resorted to patching the code instead.
-          set -e
-          FILEPATH=$GITHUB_WORKSPACE/node_modules/lighthouse/lighthouse-core/gather/gatherers/source-maps.js
-          SEARCH_TARGET="1500"
-          REPLACEMENT="10000"
-          occurrences=$(grep -c $SEARCH_TARGET $FILEPATH)
-          echo "Found $occurrences occurrences of '$SEARCH_TARGET'"
-          # Assert that there is only one match for the target string.
-          # If there are ever zero or multiple matches, the lighthouse
-          # code has changed and this logic needs to be reviewed.
-          [[ $occurrences == "1" ]]
-          # Replace the target string with the replacement string.
-          sed -i 's/$SEARCH_TARGET/$REPLACEMENT/' $FILEPATH
-          echo "Replaced '$SEARCH_TARGET' with '$REPLACEMENT' in $FILEPATH"
-        shell: bash
+      # Installing python dependencies becuase we are executing a python script for installing chrome in CI.
+      - name: Install Python deps
+        run: |
+          pip install --require-hashes --no-deps -r requirements.txt
+          pip install --require-hashes --no-deps -r requirements_dev.txt
```

Comment:
Why do we need this here? ARe we install chrome here?

Reply:
yes